config/routes.rb

```diff
@@ -3,4 +3,5 @@
 Rails.application.routes.draw do
   devise_for :users
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  resources :announcements
```

Comment:
Resources adds all actions (`new` `create` `edit` etc). If you don't need them, you can use `only:` or `except:` to filter them out